projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -723,7 +757,8 @@ class="<?php echo esc_attr( $classes['submit_button'] ); ?>"
 			</div>
 		</div>
 	</div>
-	<?php
+		<?php
+	}
```

Comment:
I think this end should be higher up before `show_subscribers_total` check, so that we keep showing that extra line if they've enabled it in the block settings.

Reply:
I think the text says something like "Join other x subscribers", that's why I left it behind

Reply:
👍 Fine for this PR but let's find a way to bring it in, with copy changes perhaps?